client/telemetry/telemetryQueriesExtended.go

```diff
@@ -77,14 +77,126 @@ func (impl *TelemetryEventClientImplExtended) getUserCreatedPluginCount() int {
 	return len(plugins)
 }
 
-func (impl *TelemetryEventClientImplExtended) getPolicyCount() int {
-	// Get global policies
-	globalPolicies, err := impl.cvePolicyRepository.GetGlobalPolicies()
+// getDeploymentWindowPolicyCount returns the count of deployment window policies
+func (impl *TelemetryEventClientImplExtended) getDeploymentWindowPolicyCount() int {
+	// TODO: Implement when deployment window policy repository is available
+	// For now, return 0 as placeholder
+	var count int
+	query := `
+		SELECT COUNT(*)
+		FROM deployment_window_policy dwp
```

Comment:
deployment_window_policy does not exist. Use column policy_of of table global_policy for this